engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -207,23 +207,20 @@ public Vector3f getViewingDirection() {
     }
 
     /**
-     * Get the gaze direction.
-     * @return the gaze direction, a quaternion.
+     * Get the viewing direction.
+     * @return the viewing direction, a quaternion.
      */
-    public Quat4f getGazeDirection() {
-        return gazeDirection;
+    public Quat4f getOrientation() {
+        return new Quat4f(viewingDirection,viewingAngle);
     }
 
     /**
-     Try to set the gaze direction. * Some cameras might be immovable, such as a VR camera, thus "request."
-     * @param gazeDirectionIn
+     Try to set the viewing direction. * Some cameras might be immovable, such as a VR camera, thus "request."
+     * @param direction
      */
-    public void requestSetGazeDirection(Quat4f gazeDirectionIn) {
-        gazeDirection = gazeDirectionIn;
-    }
-
-    public Vector3f getUp() {
-        return up;
+    public void requestSetOrientation(Quat4f direction) {
+        viewingDirection = direction.getAxis();
+        viewingAngle = direction.getAngle();
```

Comment:
Naming is better now, but I'm still not convinced this method should be here, see my reply to your comment on the matter, in an outdated commit. 

If we do go down this road I'd prefer setDesiredOrientation or perhaps suggestOrientation: requestSetOrientation can be interpreted as "request the orientation that has been set".